src/main/resources/carpet.accesswidener

```diff
@@ -7,3 +7,4 @@ accessible class net/minecraft/world/level/lighting/BlockLightSectionStorage$Blo
 accessible class net/minecraft/server/MinecraftServer$ReloadableResources
 
 accessible field net/minecraft/world/level/block/state/BlockBehaviour UPDATE_SHAPE_ORDER [Lnet/minecraft/core/Direction;
+accessible field net/minecraft/client/renderer/blockentity/ShulkerBoxRenderer model Lnet/minecraft/client/model/ShulkerModel;
```

Comment:
you could do that with a mixin - mixins are better / clearer than accessWideners which should only be used if there is no other way.

Reply:
> you could do that with a mixin - mixins are better / clearer than accessWideners which should only be used if there is no other way.



i was using a mixin....but you said dont touch vanilla code.... so i change to use accessWideners. 
i m new to mixin. if there is a way to use mixin without touch vanilla code, please tell me.

Reply:
I dug into it for a while and it seems to be successfully implemented with a mixin. But I don't know if I am using it correctly. Please review it carefully.